app/lib/screens/council_screen.dart

```diff
@@ -38,77 +77,57 @@ class _CouncilScreenState extends State<CouncilScreen> {
                         final v = value.trim();
```

Comment:
You can set the `selectedNetwork` with whatever value you change here instead of overriding the value in each case